### DISCUSSION TITLE
Rewrite usable?

### DISCUSSION THREAD

**Discussion by DevScarabyte** [FlipperPlz/BisUtils#15] [Q&A] [answered]
Is the rewrite usable atm?

**Comment by FlipperPlz** [comment, marked as answer]
Should be out by the end of the month. Currently working on all of the sublibraries at once. Focusing on P3D's right now but after that I will be focusing on polishing up the earlier implementations such as PBO. I'm currently aiming for #14 to be merged July 29th.

**Reply by DevScarabyte** [reply]
Sweet! Quick question/request. Can you make it where we can override PBO entry variables? Like manually change what the data size is or the original size? Loving the rewrite btw! Looks cleaner 😀

**Reply by FlipperPlz** [reply]
Yea I plan on adding full support for malformed objects. So far anything inheriting from StrictBinaryObject can be written/stored in a malformed manner. Validate() will ensure that the format is valid with the supplied options. The thing that will be inherently not allowed is nullability

**Reply by FlipperPlz** [reply]
Appreciate the support. Really am using this project to practice proper library design fundamentals
